### ISSUE TITLE
cmdlet Move-StaffHubTeam doesn't exist.

### ISSUE THREAD

**Issue by MarniHall** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#1925]
The cmdlet Move-StaffHubTeam doesn't exist in the powershell module listed as a prerequisite.  Without this cmdlet, none of the instructions above will result in a Staffhub team being moved to Shifts.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8f64dee1-c968-9d64-757e-7ca3263a4fc4
* Version Independent ID: a3157a8b-a127-0726-2ee4-8972e773a785
* Content: [Move your StaffHub teams to Shifts in Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/expand-teams-across-your-org/shifts/move-staffhub-teams-to-shifts-in-teams#feedback)
* Content Source: [Teams/expand-teams-across-your-org/shifts/move-staffhub-teams-to-shifts-in-teams.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/expand-teams-across-your-org/shifts/move-staffhub-teams-to-shifts-in-teams.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by jarrodmast** [comment]
yeah same here, checked the staffhub powershell site and its not even listed as a valid command!

**Comment by faron05** [comment]
I agree! It looks like an updated module is needed for the new cmdlets!

**Comment by MarniHall** [comment]
Logged a ticket with microsoft. They confirmed cmdlet not released yet.  These instructions will be valid when the powershell module update is released.  No firm ETA.

**Comment by scanum** [comment]
@officedocsbot assign @scanum

**Comment by get-itips** [comment]
Hello @LanaChin can you confirm what was told to @MarniHall ? **Thank you.**

**Comment by LanaChin** [comment]
@scanum @get-itips 

I've updated the following note at top of the article to reflect when the cmdlets will be available:

"The capability discussed in this article hasn't yet been released. It's been announced, and is coming soon, towards the middle of May 2019. If you're an admin, you can find out when this will be available in the Message Center (in the Microsoft 365 admin center)."

Thanks.

**Comment by scanum** [comment]
@MarniHall Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by scanum** [comment]
@officedocsbot close

**Comment by joshn-ridewta-com** [comment]
The note at the top is missing and so is the cmdlet.  When will the cmdlets be available?

**Comment by joshn-ridewta-com** [comment]
Clarification, it is available with version 1.0.5-alpha

**Comment by spanje** [comment]
Only 1 month left and the Move-StaffHubTeam command is still not available?!? Unable to migrate to Teams :-( When will this be available?

**Comment by joshn-ridewta-com** [comment]
I was wrong, it does, you have to be very specific about the version you install.


R. Josh Nylander
Director of IT

Whatcom Transportation Authority
4011 Bakerview Spur Road
Bellingham, WA 98226
p: 360-788-9341 / f: 360-788-9541
joshn@ridewta.com


E-mail to and from this address may be subject to public disclosure under Washington’s Public Records Act.

________________________________

**Comment by dotsndots** [comment]
Two weeks to go and trying to migrate our StaffHub.. follow the guide and find that move-staffhubteam doesn't exist in the default install module (1.02) - how do you force the install of a working module with the correct cmdlet?

**Comment by dotsndots** [comment]
Find-Module MicrosoftStaffHub -AllVersions returns one working module 1.02 - the rest including latest 1.0.6-alpha all have a System.Version error that prevent installation.

**Comment by dvnt-kaz** [comment]
This solution worked for me: https://techcommunity.microsoft.com/t5/Microsoft-StaffHub/Move-StaffHubTeam-cmdlet-not-found/m-p/849426
TLDR: Install the prerelease MicrosoftStaffHub module with PSCore6, import the module to PS and then run the Move-StaffHubTeam command.

**Comment by get-itips** [comment]
The cmdlet is present now in the last version of the module. (1.0.7)
